### ISSUE TITLE
Add birth date and address to event sign-up export

### ISSUE THREAD

**Issue by M-Kraakman** [esac-ic/esac.nl#300]
Klimco requests that birth date, address, ZIP code and city/town are added to the file when exporting sign-ups.

**Comment by ThijsEgberts** [comment]
is this allowed privacy wise? because if so I might be interested in taking this issue

**Comment by wouterbles** [comment]
I don't think this would be an issue? As the klimco can already see all data of members in the website. The export feature is probably for requesting climbing insurance stuff

**Comment by ThijsEgberts** [comment]
Oh I thought this feature would also be available to other people besides the klimco

**Comment by wouterbles** [comment]
No only the people with "management" access can export

**Comment by ThijsEgberts** [comment]
Yeah but that's also just people from committees that need to be able to make agenda items. Or is the export function a different level of management access?
